### PR TITLE
Fix typo in comment

### PR DESCRIPTION
**Pull request by longkb**
Signed-off-by: Kim Bao Long <longkb@vn.fujitsu.com>

### CODE REVIEW

File: hack/verify-shellcheck.sh [hunk 1]
```diff
@@ -178,7 +178,7 @@ fi
 # common arguments we'll pass to shellcheck
 SHELLCHECK_OPTIONS=(
   # allow following sourced files that are not specified in the command,
-  # we need this because we specify one file at at time in order to trivially
+  # we need this because we specify one file at time in order to trivially
```
Comment (johngmyers) [inline hack/verify-shellcheck.sh]:
```suggestion
  # we need this because we specify one file at a time in order to trivially
```

Comment (longkb) [reply hack/verify-shellcheck.sh]: Done :)

**Comment by rifelpet** [comment]
thanks!
/lgtm
/approve

**Pull request merged**